app/studentsApi.js

```diff
@@ -0,0 +1,32 @@
+"use strict"
+
+const express = require("express")
+const router = express.Router()
+const studentsDB = require("./studentsDB.json")
+
+
+function getStudentById(id) {
+    return studentsDB.find(student => student.id === studentId)
```

Comment:
You have forget to use id instead of studentId here:

![image](https://cloud.githubusercontent.com/assets/5324414/21112991/092548ba-c08f-11e6-9d58-c7cd88ae834c.png)